src/ucp/core/ucp_context.c

```diff
@@ -296,7 +296,9 @@ static ucs_config_field_t ucp_config_table[] = {
    ucs_offsetof(ucp_config_t, ctx.proto_enable), UCS_CONFIG_TYPE_BOOL},
 
   {"PROTO_INDIRECT_ID", "auto",
-   "Enable indirect IDs to object pointers in wire protocols",
+   "Enable indirect IDs to object pointers (endpoint, request) in wire protocols."
+   " A value of 'auto' means to enable only if error handling is enabled on the"
```

Comment:
minor: remove space before A, add \n in end of every line

Reply:
fixed